### ISSUE TITLE
idlist command not completeing, no error message

### ISSUE THREAD

**Issue by katherinehess** [samapriya/porder#32]
Hi,

I used porder a lot in May and it worked great, but now I'm coming back to it and having an issue. I tried running:
porder idlist --input "Y:/duncansongp/Planet/geojsons/amazondeg.geojson" --start "2014-01-01" --end "2019-09-04" --item "PSScene4Band" --asset "analytic" --cmax 0.1 --overlap 100 --outfile "Y:/duncansongp/Planet/orderids/amazondeg.csv"  

This same code worked quickly/ easily when I ran it in May, but now I hit enter and it doesn't fail or give me any error messages, but it never completes. 

I tried some old order and multipart download commands and those are working normally, it seems to just be idlist. 

Any idea what could be going on here?
Thanks!
Kate

**Comment by samapriya** [comment]
Hi @katherinehess 
Could you tell me your porder version you can get that by typing

```pip show porder```

and also attach your geometry and search parameters to test it out?

**Comment by samapriya** [comment]
Hi @katherinehess 
Since I have not heard back from you, I am hoping this has resolved. If not please attach the information  I requested and feel free to reopen the issue.

**Comment by katherinehess** [comment]
Hi @samapriya 
I have version 0.2.9
Here is the geometry:
[amazondeg.zip](https://github.com/samapriya/porder/files/3596347/amazondeg.zip)

And the search parameters:
dates: "2014-01-01" to "2019-09-04" (I've tried smaller date ranges and they also don't go through) 
PlanetScope 4 band, analytic
cloud max .1 (I tried it without a cloud max or overlap parameter and also didn't get anywhere)
overlap 100%

Thanks for taking a look at this!

**Comment by samapriya** [comment]
Hi @katherinehess 
Your geometry seems to be the issue here. There are line intersections and you would need to repair the geometry. I used an online tool called [mapshaper](https://mapshaper.org/) to get rid of the issues (screenshot attached). I am also attaching the correct geometry geojson file in a zip file and now it returns results as expected. I got a total of 724 images for the time period you used when I rerun the search using the correct geometry. Hope this resolves your issues, in the future I would start debugging with checking your geometry.

<img width="856" alt="geometry_repair" src="https://user-images.githubusercontent.com/6677629/64633231-d27f7880-d3c8-11e9-8b05-b3396bf546eb.PNG">


[amazondeg.zip](https://github.com/samapriya/porder/files/3596824/amazondeg.zip)
